cmd/all-in-one/main.go

```diff
@@ -89,7 +89,7 @@ func main() {
 		Use:   "jaeger-all-in-one",
 		Short: "Jaeger all-in-one distribution with agent, collector and query in one process.",
 		Long: `Jaeger all-in-one distribution with agent, collector and query. Use with caution this version
```

Comment:
"caution as this"
Does it need to be broken into 2 lines?

Reply:
just a preference to break lines longer than 100 chars.